static/src/javascripts/projects/common/modules/analytics/coreVitals.ts

```diff
@@ -1,6 +1,7 @@
 import { getCookie } from '@guardian/libs';
 import { getCLS, getFCP, getFID, getLCP, getTTFB } from 'web-vitals';
 import reportError from 'lib/report-error';
+import { shouldForceCommercialMetrics } from '../../../commercial/commercial-metrics';
```

Comment:
Now that `shouldForceMetrics` isn't used strictly for commercial metrics, I wonder if it might be more appropriate for it to be exported without the being renamed `shouldForceCommercialMetrics`, and to live somewhere other than `commercial/commercial-metrics` (maybe `common/modules/analytics/` ?).